conseil-api/src/test/scala/tech/cryptonomic/conseil/api/routes/platform/data/bitcoin/BitcoinDataOperationsTest.scala

```diff
@@ -69,7 +68,7 @@ class BitcoinDataQueriesTest
         dbHandler.run(Tables.Blocks ++= blocks).isReadyWithin(5.seconds) shouldBe true
         dbHandler.run(Tables.Transactions ++= transactions).isReadyWithin(5.seconds) shouldBe true
 
-        whenReady(sut.fetchBlocks(Query.empty)) { result =>
+        whenReady(sut.fetchTransactions(Query.empty)) { result =>
```

Comment:
Was this actually a test bug?

Reply:
Yes, it was a bug indeed. We have been testing wrong method, but the returned result was pretty much the same.